com/metacube/shoppingcart/login/LoginAsAdmin.java

```diff
@@ -0,0 +1,99 @@
+package com.metacube.shoppingcart.login;
+
+import java.util.Scanner;
+
+import com.metacube.shoppingcart.controller.ProductController;
+import com.metacube.shoppingcart.entity.Product;
+/**
+ * It will help to login as Admin
+ * to add product to list and remove and update product from list
+ * @author Vinod Patel
+ *
+ */
+public class LoginAsAdmin {
+	int choice2 = 1;
+	int choice = 0;
+	Scanner input = new Scanner(System.in);
+	ProductController productController = ProductController.getInstance();
+	public void loginAsAdmin(){
+    	while(choice2==1){
+    		System.out.println("Select The Option from the Menu to Continue : ");
+        	System.out.println("1. Add product to list.");
+        	System.out.println("2. Change product Price.");
+        	System.out.println("3. Update stock.");
```

Comment:
Do you really need this choice because stock should get auto calcalculated. When admin perform add Product and remove product then stock should get updated accordingly. I think admin not need to manage stock manually.